src/DynamoCore/Graph/Notes/NoteModel.cs

```diff
@@ -79,5 +86,147 @@ protected override void DeserializeCore(XmlElement nodeElement, SaveContext cont
         }
 
         #endregion
+
+        #region BulletSupport
+
+        /// <summary>
+        /// Method used to change the text property of the note to support bullet points.
+        /// It first personalizes the TAB spacing by converting it to space indentation. (The default spacing is too wide)
+        /// It then converts dashes to bullet characters depending on their existing indentation.
+        /// It then corrects bullet characters if their indentation has changed 
+        /// </summary>
+        /// <param name="text"> Input text with no bullet support </param>
+        /// <returns> Bullet supported text </returns>
+        private string AddBulletPointSupport(string text)
```

Comment:
I feel like this isnt great, right now we are doing all of this **every time** the text changes, which is every time a character is added to the textbox. It seems a bit excessive that we need to change the entire text every time.

Could we not simply check if the character that has been added is a tab or dash and simply convert that?

Reply:
Ok changed this so that it is tied to the characters typed